hpp-fclConfig.cmake

```
@@ -24,4 +22,13 @@ if(NOT TARGET hpp-fcl::hpp-fcl)
       INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
       INTERFACE_LINK_LIBRARIES "coal::coal"
   )
+
+  if(COAL_DISABLE_HPP_FCL_WARNINGS)
+    target_compile_definitions(
+      hpp-fcl::hpp-fcl
+      INTERFACE -DCOAL_DISABLE_HPP_FCL_WARNINGS
```

```suggestion
      INTERFACE COAL_DISABLE_HPP_FCL_WARNINGS
```

No need to add `-D`: https://cmake.org/cmake/help/latest/command/target_compile_definitions.html